_docs/_latest/configure/notifier/index.md

```diff
@@ -29,6 +29,63 @@ and varying formats to you to events in your Google Cloud Platform
 
 ## Configuring Notifier
 
+### Email Connector Config
+
+Forseti security provides an interface to add the email connector of your
+choice. A new section named `email_connector` has been introduced under 
+`notifier` as part of the new design. The connector details provided in this
+section will be used to send out inventory summary and violations via email
+when found.notify users violations when found for the enabled
+scanners. 
+
+To configure `email_connector`, follow the steps below:
+
+1. Open `forseti-security/configs/server/forseti_conf_server.yaml`.
+2. Navigate to the `notifier` > `email_connector` section.
+
+If you want the notifier to send violations and/or inventory summary via email, 
+provide valid values for all the fields mentioned below.
+
+* `auth`
+  * **Description**: The connector you want to use to receive emails.
+  * **Valid values**: String
```

Comment:
I dont think users can just put any values they want, can we also list out the list of connectors that we support at this moment? e.g. sendgrid

Reply:
Done

Reply:
I don't see `sendgrid` as a value.